src/components/PostCard/PostBodyProposal/PostBodyProposal.js

```diff
@@ -0,0 +1,140 @@
+import cx from 'classnames'
+import React, { useEffect, useState } from 'react'
+import { throttle } from 'lodash/fp'
+import { useTranslation } from 'react-i18next'
+import { useDispatch } from 'react-redux'
+import ReactTooltip from 'react-tooltip'
+import { PROPOSAL_STATUS_CASUAL, PROPOSAL_STATUS_COMPLETED, PROPOSAL_STATUS_DISCUSSION, PROPOSAL_STATUS_VOTING, PROPOSAL_TYPE_MAJORITY, PROPOSAL_TYPE_MULTI_UNRESTRICTED, PROPOSAL_TYPE_SINGLE } from 'store/models/Post'
+import {
+  addProposalVote,
+  removeProposalVote,
+  swapProposalVote
+} from '../../../store/actions/proposals'
+import QuorumBar from 'components/QuorumBar/QuorumBar'
+import './PostBodyProposal.scss'
+import RoundImageRow from 'components/RoundImageRow'
+
+const calcNumberOfVoters = (votes) => {
```

Comment:
it feels like these should be memoized values in the render that get recalculated when the votes change

Reply:
done